ocaml/xapi/cli_frontend.ml

```diff
@@ -1793,6 +1793,14 @@ let rec cmdtable_data : (string*cmd_spec) list =
       implementation=No_fd Cli_operations.sr_probe;
       flags=[];
     };
+    "sr-probe-ext",
+    {
+      reqd=["type"];
+      optn=["host-uuid";"device-config:";"sm-config:"];
```

Comment:
Is the semicolon at the end of `device-config` and `sm-config` expected?

Reply:
Yes, I copied these things from the `sr-probe` definition, I think that means these are set parameters.